### PR TITLE
docs: New governance information and About Carbon edits

### PR DESCRIPTION
**Pull request by connor-leech**
Expanded About Carbon with additional Governance information.

### CODE REVIEW

File: src/pages/getting-started/about-carbon.mdx [hunk 1]
```diff
@@ -1,73 +1,90 @@
...
+## Governance
 
-It's a distributed effort, guided by the principles of the [open-source movement](https://en.wikipedia.org/wiki/Open-source-software_movement). Carbon's users are also its makers, and everyone is encouraged to contribute.
+The Carbon team encourages adoption through better guidance, training, community development, maintenance, and support. The system is primarily maintained by a core team and governed by an executive board of design advisors.
 
-#### Carbon is inclusive
+### Adoption
 
-It's designed and built to be accessible to all, regardless of ability or situation.
+**We conduct training classes and offer certifications.** These cover introductions to Carbon and participating in this community. Each program is piloted internally and available upon release externally. We teach at conferences, bootcamp labs, and wherever else we're welcome.
```
Comment (janchild) [inline src/pages/getting-started/about-carbon.mdx]:
```suggestion
**We conduct training classes and offer certifications.** These cover introductions to Carbon and participating in this community. Each program is piloted internally and available upon release externally. We teach at conferences, bootcamp labs, and wherever else we're needed.
```

File: src/pages/getting-started/about-carbon.mdx [hunk 2]
```diff
@@ -1,73 +1,90 @@
...
+**Carbon is modular and flexible.** The system's modularity ensures maximum flexibility in execution. Its components are designed to work seamlessly with each other, in whichever combination suits the needs of the user.
+
+**Carbon puts the user first.** Using rigorous research into users' needs and desires, Carbon is laser-focused on real people.
+
+**Carbon builds consistency.** Based on the IBM Design Language, every element and component of Carbon was designed from the ground up to work elegantly together to ensure consistent, cohesive user experiences.
+
+## Governance
 
-It's a distributed effort, guided by the principles of the [open-source movement](https://en.wikipedia.org/wiki/Open-source-software_movement). Carbon's users are also its makers, and everyone is encouraged to contribute.
+The Carbon team encourages adoption through better guidance, training, community development, maintenance, and support. The system is primarily maintained by a core team and governed by an executive board of design advisors.
```
Comment (janchild) [inline src/pages/getting-started/about-carbon.mdx]:
```suggestion
The Carbon team encourages adoption through guidance and training, community development, maintenance, and support. The system is primarily maintained by a core team and governed by an executive board of design advisors.
```

File: src/pages/getting-started/about-carbon.mdx [hunk 3]
```diff
@@ -1,73 +1,90 @@
...
-It's designed and built to be accessible to all, regardless of ability or situation.
+**We conduct training classes and offer certifications.** These cover introductions to Carbon and participating in this community. Each program is piloted internally and available upon release externally. We teach at conferences, bootcamp labs, and wherever else we're needed.
 
-#### Carbon is modular and flexible
+**We engage the community.** We strive to be one of the world’s best design systems and we’re always [open to feedback](https://github.com/carbon-design-system/carbon-website/issues/new?assignees=&labels=feedback&template=feedback.md&title=). We communicate with you about Carbon's state and roadmap through GitHub support, blog posts, and Twitter.
 
-Carbon's modularity ensures maximum flexibility in execution. Its components are designed to work seamlessly with each other, in whichever combination suits the needs of the user.
+**Contributed components and patterns include a list of maintainers.** The Carbon team triages and supports maintenance requests that do not have a maintainer. Unmaintained patterns will be demoted to Experimental if the Carbon team decides not to maintain them.
```
Comment (connor-leech) [inline src/pages/getting-started/about-carbon.mdx]:
```suggestion
**Contributed components and patterns include a list of maintainers.** The Carbon team triages and supports maintenance requests that do not have a maintainer. Unmaintained patterns will be demoted to Experimental if the Carbon team decides not to maintain them.

**We provide support for users of the design system.** The Carbon team engages with users primarily through GitHub. A member of the Carbon team responds to all issues and pull requests.
```

File: src/pages/getting-started/about-carbon.mdx [hunk 4]
```diff
@@ -1,73 +1,92 @@
...
+- [Vue](https://github.com/carbon-design-system/carbon-components-vue): Community maintained
 
-## Who is Carbon for?
+**Design patterns are harvested from products built with Carbon.** These become part of the design system. Teams can can use these well-defined patterns in their work and contribute patterns back to the system.
 
-#### Designers
+### Contribution acceptance
 
-Carbon is the official implementation of the IBM Design Language for product and web designers, and represents an ever-growing ecosystem of design assets and guidance. With a comprehensive set of human interface guidelines, design kits, and documentation, Carbon helps designers work faster and smarter.
+We bring experimental components and patterns into the design library as quickly as possible with little barrier to entry. Each pattern will be badged with a level of acceptance.
```
Comment (connor-leech) [inline src/pages/getting-started/about-carbon.mdx]:
```suggestion
We bring experimental components and patterns into the design library as quickly as possible with little barrier to entry. ```

**Comment by connor-leech** [comment]
Mistakes were made. New version here: https://github.com/carbon-design-system/carbon-website/pull/155

**Pull request closed without merging**